samples/StaticHtmlComposites/README.md

```diff
@@ -1,8 +1,8 @@
-# Basic sample for using CallComposite and ChatComposite on a plain HTML page
+# Basic sample for using UI Composites on a plain HTML page
 
 This sample shows how to embed the ready-to-use Calling and Chat composite components on your plain HTML page with a `<script>` tag. This is for you if your web app is not a React application but you still want to use our React components. The React parts are being hidden away behind helper functions inside a JavaScript bundle. The sample consists of the following files:
 
-- `composites.js` exports two basic helper functions that use React to render the `CallComposite` or `ChatComposite` into a container HTML element.
+- `composites.js` exports two basic helper functions that use React to render the UI Composite into a container HTML element.
```

Comment:
I guess we separated entry code into 3 different files, also it's not 2 helper functions